### PR TITLE
pastsub_replacement

### LINKED ISSUE #40: Deployment fails with multiple script commands
In the past, I could use my `drone.yml` file  like this:

```yaml
- script: cd app
- script: docker-compose up -d
```

Which Drone would interpret as:

```bash
> cd app && docker-compose up -d
```

However, since the latest version, it now interprets it as:

```bash
> cd app docker-compose up -d
```
breaking the deployment

### PR DESCRIPTION
**Pull request by tomeli5n**
fixes #40 , issue related to Bash 5.2 and '&' as delimitier.

### CODE REVIEW

**Comment by fumoretti** [comment]
Builded localy with your patch and it works! Thank You!

**Comment by mjwwit** [comment]
Thanks for this!
